docs/tools.md

```diff
@@ -60,7 +60,7 @@ Flags:
   -default-org-id int
     	default org ID to send to downstream services if none is provided (default -1)
   -discard-prefixes string
-    	discard data points starting with one of the given prefixes separated by | (may be given multiple times, once per topic, as a comma-separated list)
+    	discard data points starting with one of the given prefixes separated by | (may be given multiple times, once per topic specified in 'metrics-topic', as a comma-separated list)
```

Comment:
These flags were a little confusing to me, I didn't understand how you could map to a topic using comma separated values